metadata/src/main/java/org/apache/kafka/metadata/migration/KRaftMigrationDriver.java

```diff
@@ -458,8 +458,9 @@ public void run() throws Exception {
                 zkMigrationClient.readAllMetadata(batch -> {
                     try {
                         if (log.isTraceEnabled()) {
-                            log.trace("Migrating {} records from ZK: {}", batch.size(), recordBatchToString(batch));
+                            log.info("Migrating {} records from ZK: {}", batch.size(), recordBatchToString(batch));
                         } else {
+                            log.info("Migrating {} records from ZK: {}", batch.size(), recordBatchToString(batch));
```

Comment:
This looks like a typo. Why is this the same as in the trace-enabled case?

Reply:
This file shouldn't have been part of this commit at all. Thanks for catching this.